ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -660,6 +660,7 @@ ReplicaMetadataResponse getReplicaMetadataResponse(List<RemoteReplicaInfo> repli
       logger.trace("Remote node: {} Thread name: {} Remote replica: {} Key from remote: {}", remoteNode, threadName,
           remoteReplicaInfo.getReplicaId(), storeKey);
       StoreKey convertedKey = storeKeyConverter.getConverted(storeKey);
+      logger.trace("SkipPredicate: {}", skipPredicate);
```

Comment:
Do we have a `toString()` for `skipPredicate`?

Reply:
Updated.